matrix/rules.py

```diff
@@ -313,6 +313,15 @@ def exception_handler(self, context, loop=None, exc_ctx=None):
             return
         self._reported = True
 
+    async def model_change(self, delta, old_obj, new_obj, juju_model):
+        self.bus.dispatch(
```

Comment:
We should devote a few minutes to thinking if libjuju would benefit from having something like bus.py included in it.